libphobos/src/std/file.d

```diff
@@ -3097,11 +3306,9 @@ void copy(RF, RT)(auto ref RF from, auto ref RT to, PreserveAttributes preserve
     copy!Types(from, to, preserve);
 }
 
-unittest // issue 15319
+@safe unittest // issue 15319
 {
-    import std.file : dirEntries;
-    auto fs = dirEntries(getcwd, SpanMode.depth);
-    assert(__traits(compiles, copy(fs.front, fs.front)));
+    assert(__traits(compiles, copy("from.txt", "to.txt")));
```

Comment:
This is the only part that has been taken from a latter release (in 2.072, it was replace with `dirEntries("/tmp")`, which fails when you don't have privileges to traverse all directories.